README.md

```diff
@@ -41,20 +41,14 @@ These extensions will likely become mandatory later.
 ### AMD (RADV)
 
 For AMD, RADV is the recommended driver and the one that sees most testing on AMD GPUs.
-The recommendation here is to use a driver built from Git.
+The recommendation here is at least Mesa 22.0 since it supports `VK_KHR_dynamic_rendering`.
```

Comment:
It's not really a recommendation if older drivers don't work, is it?

Reply:
True :D Fixed.